src/style.css

```diff
@@ -4,6 +4,11 @@
 
   --bs-secondary: #fff1bf;
   --bs-secondary-rgb: 255, 241, 191;
+
+  --swiper-pagination-color: #ffa500;
+  --swiper-pagination-bullet-size: 13px;
```

Comment:
Minor point, but for future, better to prefer usage of even numbers instead of odd.
Also, better practice is to use numbers that devided by 8. 

FYI: @ashotvdovychenko